components/builder-web/app/shared/project-settings/project-settings.component.html

```diff
@@ -56,7 +56,7 @@ <h3 class="plan-actions">Actions</h3>
                 <ul>
                     <li *ngFor="let installation of installations">
                         <a (click)="selectInstallation(installation)" [class.active]="installation === selectedInstallation">
-                            {{ installation.get("id") }}
+                            {{ installation.get("account").get("login") }}
```

Comment:
You could also use the slightly briefer and more forgiving form `installation.getIn(["account", "login"])`, but this totally works, too. 😄  (In this case, we know we'll always have a `login` on `account`, but when you aren't sure of that, `getIn` will prevent the UI from blowing up on the call to `.get` on `undefined`).

Reply:
Thanks for pointing that out!  I've updated it.